core/src/ast/operator.rs

```diff
@@ -59,6 +60,7 @@ impl ToSql for BinaryOperator {
             BinaryOperator::And => "AND".to_owned(),
             BinaryOperator::Or => "OR".to_owned(),
             BinaryOperator::Xor => "XOR".to_owned(),
+            BinaryOperator::ShiftLeft => "<<".to_owned(),
```

Comment:
though it may look too long name, but it would be important to mention that the operation is "bitwise".
so it would be better to use explicit name: `BitwiseShiftLeft`

ref. https://github.com/gluesql/gluesql/pull/1281